<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TradeOperationPanel</class>
 <widget class="QWidget" name="TradeOperationPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>7</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="LabelStockCode">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>股票缩写</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelStockName">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>股票全称</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>7</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="ButtonBuy">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>买入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonSell">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>卖出</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonBuyEmpty">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>沽空</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonSellEmpty">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>平仓</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="Radio2">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>梭哈</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioAll">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/2仓</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Radio3">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/3仓</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Radio4">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1/4仓</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>7</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="TabRevoke">
      <attribute name="title">
       <string>撤单</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTableView" name="TableRevoke"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPosition">
      <attribute name="title">
       <string>持仓</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTableView" name="TablePosition"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
